docs/en/observability/apm/agent-server-compatibility.asciidoc

```diff
@@ -24,7 +24,7 @@ The chart below outlines the compatibility between different versions of Elastic
 |`1.x` |≥ `8.12`
 
 // Java
-.1+|**Java agent**
+.1+|**Java agent**{empty}footnote:[There is a known issue for some versions of the Java agent. Read more in <<apm-empty-metricset-values,Known issues>>.]
```

Comment:
Can we be more specific here? Something like

.1+|**Java agent**{empty}footnote:[Java agent < 1.43.0 not fully compatible with APM Server >= 8.11.0. Read more in <<apm-empty-metricset-values,Known issues>>.]